packages/ui/src/components/molecules/attachment/useAttachment.ts

```
@@ -12,24 +12,55 @@ export interface AttachmentProps {
-  const fileUploadMutation = UsePostFileUpload();
+  const fileUploadMutation = UsePostUploadfile({
+    onSuccess: (res) => {
+      const ids: number[] = [];
+      const data = res.data;
+      if (data.length) {
+        data.forEach((item) => {
+          ids.push(item.id);
+        });
+        setFiles((prev) =>
+          prev.map((file, index: number) => {
+            delete file.loading;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)